cg/store/models.py

```diff
@@ -1002,29 +1002,29 @@ class IlluminaFlowCell(RunDevice):
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
 
-class RunMetrics(Base):
-    """Model for storing run devices."""
+class InstrumentRun(Base):
+    """Model for storing instrument runs."""
```

Comment:
It would be good to add in the docstring which are the parents for a new reader
```suggestion
    """Parent model for instrument runs."""
```